pages/advanced/content/refs.md

```diff
@@ -14,7 +14,8 @@ reference.
 var x = 'normal code block...'
 ```
 
-```interactive {clickToRender: true, summary: 'An input in a form'}
+```
+interactive {clickToRender: true, summary: 'An input in a form'}
```

Comment:
I understand what `interactive {clickToRender: true, summary: 'An input in a form'}` is and I will write method ignore the lang.